src/main/java/com/amazonaws/kvstranscribestreaming/KVSTranscribeStreamingLambda.java

```diff
@@ -281,7 +281,12 @@ private void closeFileAndUploadRawAudio(KVSStreamTrackObject kvsStreamTrackObjec
         } else {
             logger.info("Skipping upload to S3.  saveCallRecording was disabled or audio file has 0 bytes: " + kvsStreamTrackObject.getSaveAudioFilePath().toString());
         }
-
+        //Delete raw files from the lambda function
+        File fileToDelete = new File(kvsStreamTrackObject.getSaveAudioFilePath().toString());
+        if (fileToDelete != null) {
+            logger.info("Raw file deleted");
+            fileToDelete.delete();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The result of deleting a file on this line should be checked and failure should be logged. java.io.File.delete() will simply return false if it fails to delete the file, for example, when the file does not exist. It will not throw unless there is a security exception.